### PR TITLE
feat(call-pill): add default toggled state - vue3

### PR DESCRIPTION
**Pull request by fabiangarga**
# PR Title

## :hammer_and_wrench: Feat

- [ ] Fix
- [x] Feature
- [ ] Refactoring
- [ ] Documentation

## :book: Description

Add a default behavior to control the default state of the collapsible component

## :bulb: Context

For some of our implementation, there is a need to render the call pill with the default state of the collapsible component being open

## :pencil: Checklist

- [x] I have reviewed my changes
- [x] I have added tests
- [x] I have added all relevant documentation
- [ ] I have validated components with a screen reader
- [ ] I have validated components keyboard navigation
- [x] I have considered the performance impact of my change
- [x] I have checked that my change did not significantly increase bundle size
- [ ] I am exporting any new components or constants in the index.js in the component directory
- [ ] I am exporting any new components or constants in the index.js in the root

## :camera: Screenshots / GIFs

https://github.com/dialpad/dialtone-vue/assets/5950950/93a0298c-faaa-4205-9b2c-38220467d77c

### CODE REVIEW

File: recipes/conversation_view/feed_pill/feed_item_pill.stories.js [hunk 1]
```diff
@@ -57,45 +63,25 @@ const DefaultTemplate = (args) => createTemplateFromVueFile(
   DtRecipeFeedItemPillDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
+  DtRecipeFeedItemPillVariantsTemplate,
+);
```
Comment (juliodialpad) [inline recipes/conversation_view/feed_pill/feed_item_pill.stories.js]: Remove argTypes here while moving to Vue 3 or the story might not work properly, check other stories for references.

```suggestion
const VariantsTemplate = (args) => createTemplateFromVueFile(
  args,
  DtRecipeFeedItemPillVariantsTemplate,
);
```

File: recipes/conversation_view/feed_pill/feed_item_pill_default.story.vue [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-    :aria-label="$attrs.ariaLabel"
-    :toggleable="$attrs.toggleable"
+    :icon-name="iconName"
+    :title="title"
+    :wrapper-class="wrapperClass"
+    :button-class="buttonClass"
+    :border-color="borderColor"
+    :aria-label="ariaLabel"
+    :toggleable="toggleable"
+    :default-toggled="defaultToggled"
```
Comment (juliodialpad) [inline recipes/conversation_view/feed_pill/feed_item_pill_default.story.vue]: This properties need to be `$attrs.*` if they're coming from `.stories.js` on Vue 3 version.

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Small issues detected, please fix

File: recipes/conversation_view/feed_pill/feed_item_pill.stories.js (hunk 1 above)
Comment (fabiangarga) [reply recipes/conversation_view/feed_pill/feed_item_pill.stories.js]: Oh wow, didn't saw this one, thanks!

**Review by juliodialpad — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**